src/general/OptionsMenu.tscn

```diff
@@ -477,6 +478,11 @@ layout_mode = 2
 size_flags_horizontal = 0
 text = "DISPLAY_BACKGROUND_DISTORTION_EFFECT"
 
+[node name="LowQualityBlur" type="CheckBox" parent="CenterContainer/VBoxContainer/VBoxContainer/VBoxContainer/PanelContainer/Graphics/ScrollContainer/MarginContainer/VBoxContainer"]
+layout_mode = 2
+size_flags_horizontal = 0
+text = "LOW_QUALITY_BACKGROUND_BLUR"
```

Comment:
Maybe this could do with a tooltip? To explain turning this on improves performance by using a lower resolution blurring for the background (when blur is used).